tests/installation/bootloader_svirt.pm

```diff
@@ -269,6 +269,8 @@ sub run {
         # does not work.
         if (my $bridges = $svirt->get_cmd_output("virsh iface-list --all | grep -w active | awk '{ print \$1 }' | tail -n1 | tr -d '\\n'")) {
             $ifacecfg{type} = 'bridge';
+            # Due to poo#126647, the default iface 'ovs-system' on xen platform can not work fine, so we need to use br0 instead
+            $bridges = 'br0' if $bridges eq 'ovs-system';
```

Comment:
This would additional require the setup of br0 on the host? How would make it ensured, by salt?

Reply:
Good question, on our xen host, br0 is pre-configure there: 

```#cat ifcfg-br0
STARTMODE='auto'
BOOTPROTO='dhcp'
OVS_BRIDGE='yes'
OVS_BRIDGE_PORT_DEVICE_1='eth0'
```

Reply:
As xen currently isn't salted properly anyway I add it to https://progress.opensuse.org/issues/128222 to remind us this is covered